scripts/start_quickstart_argilla.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e
+
+whoami
+
+# Generate hashed passwords
+team_password=$(htpasswd -nbB "" "$TEAM_PASSWORD" | cut -d ":" -f 2 | tr -d "\n")
+argilla_password=$(htpasswd -nbB "" "$ARGILLA_PASSWORD" | cut -d ":" -f 2 | tr -d "\n")
+
+# Add data to users.yml file
+echo "Creating users schema"
+cat >"$HOME"/users.yml <<EOF
+- username: "team"
+  api_key: $TEAM_API_KEY
+  full_name: Team
+  email: team@argilla.io
+  hashed_password: $team_password
+  workspaces: []
+
+- username: "argilla"
+  api_key: $ARGILLA_API_KEY
+  full_name: Argilla
+  email: argilla@argilla.io
+  hashed_password: $argilla_password
+  workspaces: ["team"]
+EOF
+
+# Start Elasticsearch
+echo "Starting Elasticsearch"
+elasticsearch 1>/dev/null 2>/dev/null &
+
+echo "Waiting for elasticsearch to start"
+sleep 15
+
+# Load data
+if [ "$LOAD_DATA_ENABLE" == "true" ]; then
```

Comment:
@rhoitjadhav as discussed. Let's change this variable to `LOAD_DATASETS` and use the following options:

1. `full`: it loads all current datasets from load_data.py
2. `single`: it loads just the `sst-sentiment-explainability` (that is until line 53 of `load_data.py`
3. `none`: it doesn't load any dataset

Reply:
💯 !